network-api/networkapi/wagtailpages/migrations/0016_merge_20201116_2028.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.16 on 2020-11-16 20:28
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailpages', '0015_auto_20201110_1957'),
+        ('wagtailpages', '0015_buyersguidepage'),
+    ]
+
+    operations = [
+    ]
```

Comment:
Going out on a limb and suggest manually merging 0016-0018 into a single file.

Reply:
Done!